.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -0,0 +1,91 @@
+name: Report a bug 🐛
+description: Create a report to help us improve
+labels: ["bug"]
+body:
+- type: textarea
+  id: description
+  attributes:
+    label: Report
+    description: "What bug have you encountered?"
+    placeholder: "A clear and concise description of what the bug is."
+- type: textarea
+  id: expected-behavior
+  attributes:
+    label: Expected Behavior
+    description: What did you expect to happen?
+    placeholder: What did you expect to happen?
+  validations:
+    required: true
+- type: textarea
+  id: actual-behavior
+  attributes:
+    label: Actual Behavior
+    description: Also tell us, what did you see is happen?
+    placeholder: Tell us what you see that is happening
+  validations:
+    required: true
+- type: textarea
+  id: repro-steps
+  attributes:
+    label: Steps to Reproduce the Problem
+    description: "How can we reproduce this bug? Please walk us through it step by step."
+    value: |
+      1.
+      2.
+      3.
+  validations:
+    required: true
+- type: textarea
+  id: logs
+  attributes:
+    label: Logs from KEDA HTTP operator
+    description: "Provide logs from the KEDA HTTP operator, if need be."
+    value: |
+      ```
+      example
+      ```
+  validations:
+    required: false
+- type: input
+  id: keda-http-version
+  attributes:
+    label: What version of the KEDA HTTP Addon are you running?
+  validations:
+    required: false
+- type: dropdown
+  id: kubernetes-version
+  attributes:
+    label: Kubernetes Version
+    description: What version of Kubernetes that are you running?
+    options:
+    - "1.22"
```

You are behind my friend :D
```suggestion
    - "1.23"
    - "1.22"
```